content/docs/command-reference/remote/modify.md

```diff
@@ -582,6 +582,69 @@ more information.
 
 </details>
 
+<details>
+
+### Click for WebDAV
+
+- `token` - token for WebDAV server, can be empty in case of using
+  `user/password` authentication.
+
+  ```dvc
+  $ dvc remote modify --local myremote token mytoken
+  ```
+
+- `user` - username for WebDAV server, can be empty in case of using `token`
+  authentication. The order in which DVC searches for username is:
+
+  1. `user` specified in one of the DVC configs;
+  2. `user` specified in the url (e.g. `webdav://user@example.com/path`)
+
+  ```dvc
+  $ dvc remote modify --local myremote user myuser
+  ```
+
+- `password` - password for WebDAV server, can be empty in case of using `token`
+  authentication.
+
+  ```dvc
+  $ dvc remote modify --local myremote password mypassword
+  ```
+
+> The username, password, and token (may) contain sensitive user info.
+> Therefore, it's safer to add them with the `--local` option, so they're
+> written to a Git-ignored config file.
+
+- `ask_password` - ask each time for the password to use for `user/password`
+  authentication.
+
+  ```dvc
+  $ dvc remote modify myremote ask_password true
+  ```
+
+  > Note that the `password` parameter takes precedence over `ask_password`. If
+  > `password` is specified, DVC will not prompt the user to enter a password
+  > for this remote.
+
+- `cert_path` - path to certificate used for WebDAV server authentication.
+
+  ```dvc
+  $ dvc remote modify myremote cert_path /path/to/cert
+  ```
+
+- `key_path` - path to private key to use to access a remote.
+
+  ```dvc
+  $ dvc remote modify myremote key_path /path/to/key
+  ```
```

Comment:
Are these alternatives to `token` and to `user/password` auth?

Reply:
No, according to the WebDAV client documentation (and that is also what I can infer from the implementation), these are additional parameters. But they are optional parameters, probably we should give this information.

Reply:
No need to say they're optional. Most parameters except what's shown in the `remote add` reference are assumed to be optional. But ideally we should briefly explain what these are for, or link to some WebDAV documentation.

Reply:
I have added some explanation, is it enough? I don't know what I could link to, these are not really WebDAV specific options - they are just passed to the underlying [`requests`](https://requests.readthedocs.io/en/master/) API.

WebDAV seems to be the only remote having such options, even though `HTTP` is also based on `requests` it does not expose these option?

Reply:
Yep, that was more than enough. Thanks @iksnagreb!

Reply:
> WebDAV seems to be the only remote having such options, even though HTTP is also based on requests it does not expose these option?

Good question. TBH Idk but maybe @efiop, who I always pick on for these questions 😬, does.